snippets/ev3/ps4/main.py

```diff
@@ -0,0 +1,139 @@
+            print("The Triangle button was released")
+        elif code == 307 and value == 1:
+            print("The Triangle button was pressed")
+
+        # React to the Square button
+        elif code == 308 and value == 0:
+            print("The Square button was released")
+        elif code == 308 and value == 1:
+            print("The Square button was pressed")
+
+        # React to the L1 button
+        elif code == 310 and value == 0:
+            print("The L1 button was released")
+        elif code == 310 and value == 1:
+            print("The L1 button was pressed")
+
+        # React to the R1 button
+        elif code == 311 and value == 0:
+            print("The R1 button was released")
+        elif code == 311 and value == 1:
+            print("The R1 button was pressed")
+
+        # React to the L2 button
+        elif code == 312 and value == 0:
+            print("The L2 button was released")
+        elif code == 312 and value == 1:
+            print("The L2 button was pressed")
+
+        # React to the R2 button
+        elif code == 313 and value == 0:
+            print("The R2 button was released")
+        elif code == 313 and value == 1:
+            print("The R2 button was pressed")
+    
+    elif ev_type == 3: # Stick was moved
+
+        # The sticks often trigger non-stop event, comment this out if you are not using the sticks as 
+        # part of your project, or it becomes hard to read other data
+        
+        # React to the left stick vertical 
+        if code == 1:
+            print("The left stick vertical is at ",value)
+            left_speed = scale(value, (0,255), (100, -100))
+        
+        # React to the left stick horizontal 
+        elif code == 0:
+            print("The left stick horizontal is at ",value)
+
+        # React to the right stick vertical
+        elif code == 4:
+            print("The right stick vertical is at ",value)
+            right_speed = scale(value, (0,255), (100, -100))
+
+        # React to the right stick horizontal
+        elif code == 3:
+            print("The right stick horizontal is at ",value)
+        
+
+        # React to the Directional pad
+        if code == 16 and value == -1:
+            print("The horizontal directional pad is left")
+        elif code == 16 and value == 1:
+            print("The horizontal directional pad is right")
+        elif code == 16 and value == 0:
+            print("The horizontal directional pad is released")
+
+        elif code == 17 and value == -1:
+            print("The vertical directional pad is up")
+        elif code == 17 and value == 1:
+            print("The horizontal directional pad is down")
+        elif code == 17 and value == 0:
+            print("The horizontal directional pad is released")
+
+    # Set motor speed
+    left_motor.dc(left_speed)
+    right_motor.dc(right_speed)
+
+    # Read the next event
+    event = in_file.read(EVENT_SIZE)
+
+in_file.close()
```

Comment:
Could you add an empty new line?

You can have VSCode do this automatically for you:

![image](https://user-images.githubusercontent.com/12326241/87879606-42e52600-c9ec-11ea-9455-b16101f2347e.png)

Reply:
And finally, can you change the commit message to something like this to be consistent with the existing log?

snippets/ev3: add PS4 snippet